src/main/java/cms/gov/madie/measure/resources/MeasureTransferController.java

```diff
@@ -196,4 +201,38 @@ private void setMeasureElmJsonAndErrors(Measure measure, String apiKey, String h
       measure.setCqlErrors(true);
     }
   }
+
+  protected List<Group> reorderGroupPopulations(List<Group> groups) {
+    List<Population> newPopulations;
+    List<Group> newGroups = new ArrayList<>();
+    if (!CollectionUtils.isEmpty(groups)) {
+      for (Group group : groups) {
+        newPopulations = new ArrayList<>();
+        List<Population> populations = group.getPopulations();
+        if (!CollectionUtils.isEmpty(populations)) {
+          newPopulations.add(findPopulation(populations, PopulationType.INITIAL_POPULATION));
+          newPopulations.add(findPopulation(populations, PopulationType.DENOMINATOR));
+          newPopulations.add(findPopulation(populations, PopulationType.DENOMINATOR_EXCLUSION));
+          newPopulations.add(findPopulation(populations, PopulationType.NUMERATOR));
+          newPopulations.add(findPopulation(populations, PopulationType.NUMERATOR_EXCLUSION));
+          newPopulations.add(findPopulation(populations, PopulationType.DENOMINATOR_EXCEPTION));
+          group.setPopulations(newPopulations);
+          newGroups.add(group);
+        }
+      }
+    }
+    return newGroups;
```

Comment:
TestCaseServiceUtil.EXPECTED_VALUE_ORDER has more than these 6 types of populations, With the above logic, we will be loosing those additional populations.

Reply:
We need to take all those PopulationTypes into account while reordering. I believe we should be sorting the collection based on this list.

Reply:
Sorting out list to find a match itself doesn't look like a viable solution.

Reply:
@RohitKandimalla I think you are right, measure population and measure population exclusion should be added for CV. Observations are in a different field so no need to add them.

Reply:
So for a Cohort and Ratio measures we are adding all 6 populations ? Looks like we are not necessarily reordering the existing populations but we are adding new ones too.

Reply:
Also lets not mutate groups object

Reply:
@RohitKandimalla Just did the changes. Thanks!